tests/Utilities/MakeWithRandomValues.hpp

```diff
@@ -70,6 +70,53 @@ struct FillWithRandomValuesImpl<Variables<tmpl::list<Tags...>>,
 /// \endcond
 }  // namespace TestHelpers_detail
 
+/// \ingroup TestingFrameworkGroup
+/// \brief A uniform distribution functor for which the behavior can be easily
+/// customized
+///
+/// \details The two constructors take either a low and high bound arguments or
+/// a similar 2-component array , which can be passed in in any type from which
+/// the generated type can be constructed. The default generation function
+/// operates as would be expected for a double
+template <typename T>
+struct uniform_custom_distribution {
+  using result_type = T;
+  std::array<T, 2> bounds;
+  template <typename Bound>
+  uniform_custom_distribution(Bound Low, Bound High) noexcept {
+    bounds = std::array<T, 2>{{T(Low), T(High)}};
+  }
+  template <typename Bound>
+  explicit uniform_custom_distribution(std::array<Bound, 2> bounds) noexcept
+      : bounds{{T(bounds[0]), T(bounds[1])}} {}
+
+  template <class Generator>
+  T operator()(Generator& g) noexcept {
+    return ((T(g()) - Generator::min()) *
```

Comment:
Why all this min() and max()?

Reply:
It's part of the specification for distributions given a random bit generator.
[cppreference - UniformBitGenerator](https://en.cppreference.com/w/cpp/named_req/UniformRandomBitGenerator)